consensus/types/src/light_client_update.rs

```diff
@@ -403,6 +404,31 @@ impl<E: EthSpec> LightClientUpdate<E> {
         }
         true
     }
+
+    // TODO(lc-update) does this math look correct?
```

Comment:
Yes looks good, you can link the spec and that mention that an update has two headers